akvo/rsr/models/report.py

```diff
@@ -0,0 +1,42 @@
+# -*- coding: utf-8 -*-
+
+# Akvo RSR is covered by the GNU Affero General Public License.
+# See more details in the license.txt file located at the root folder of the Akvo RSR module.
+# For additional details on the GNU license please see < http://www.gnu.org/licenses/agpl.html >.
+
+
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from ..fields import ValidXMLCharField, ValidXMLTextField
+
+
+class ReportFormat(models.Model):
+    name = ValidXMLCharField(_('name'), max_length=30, unique=True)
```

Comment:
`ReportFormat.name` or `format.name` seemed better to me than `format.format` or `ReportFormat.format`.  I'm leaning towards leaving `name` as it is. It may just be me, and if you insist, I can change this.